content/03.methods.md

```diff
@@ -27,10 +27,39 @@ We initially applied the beta-mixture quantile normalization (BMIQ) method [@doi
 We report uncorrected results in the main paper for simplicity.
 
 To make a fair comparison in each of the experiments displayed in the results, we used the intersection of TCGA samples having measurements for all of the datasets being compared in that experiment.
-This resulted in 3 distinct sets of samples: 9,074 samples shared between expression and mutation data, 7,981 samples shared between expression/mutation/27K methylation/450K methylation, and 5,282 samples shared between expression/mutation/27K methylation/450K methylation/RPPA/mutational signatures.
+This resulted in 3 distinct sets of samples: 9,074 samples shared between {expression, mutation} data, 7,981 samples shared between {expression, mutation, 27K methylation, 450K methylation}, and 5,282 samples shared between {expression, mutation, 27K methylation, 450K methylation, RPPA, mutational signatures}.
 When we dropped samples between experiments as progressively more data types were added, we observed that the dropped samples had approximately the same cancer type proportions as the dataset as a whole.
 In other words, samples that were profiled for one data type but not another did not tend to come exclusively from one or a few cancer types.
 Exceptions included acute myeloid leukemia (LAML) which had no samples profiled in the RPPA data, and ovarian cancer (OV) which had only 8 samples with 450K methylation data.
 More detailed information on cancer type proportions profiled for each data type is provided in (the supplement).
 
-(Include Venn diagrams of samples in supplement)
+### Training classifiers to detect cancer mutations
+
+We trained logistic regression classifiers to predict whether or not a given sample has a mutational event in a given target gene, using data from various -omics datasets as explanatory variables.
+As target genes, we considered 124 genes from the cancer-related gene dataset described in Vogelstein et al. 2013 [@doi:10.1126/science.1235122].
```

Comment:
I guess you probably considered 3 sets: vogelstein genes, most mutated genes, randomly selected genes.

Reply:
Right, good catch - added more detail in 31d05c3